From 1bb611fb92d3d689766fbe43037c6b0620acd219 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Wed, 6 Mar 2024 18:02:12 +0100
Subject: [PATCH 0718/1695] dts: stm32: rename IOM EVAL pins muxing on
 stm32mp257f-ev1

Rename IOM EVAL pins muxing configurations on stm32mp257f-ev1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1e21c0b1cc10c8aead1ba5f9056336e8a8a2ba17 dts: stm32: rename IOM EVAL pins muxing on stm32mp257f-ev1]
Change-Id: I63a3d8641364238997cbdcccc3bd280020e3a817
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/366666
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 1a00f0c0d..76a48331d 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -345,12 +345,12 @@
 	memory-region = <&mm_ospi1>;
 	memory-region-names = "mm_ospi1";
 	pinctrl-names = "default", "sleep";
-	pinctrl-0 = <&ospi1_clk_pins_a
-		     &ospi1_io03_pins_a
-		     &ospi1_cs0_pins_a>;
-	pinctrl-1 = <&ospi1_clk_sleep_pins_a
-		     &ospi1_io03_sleep_pins_a
-		     &ospi1_cs0_sleep_pins_a>;
+	pinctrl-0 = <&ospi_port1_clk_pins_a
+		     &ospi_port1_io03_pins_a
+		     &ospi_port1_cs0_pins_a>;
+	pinctrl-1 = <&ospi_port1_clk_sleep_pins_a
+		     &ospi_port1_io03_sleep_pins_a
+		     &ospi_port1_cs0_sleep_pins_a>;
 	status = "okay";
 };
 
-- 
2.47.2

